gdcdatamodel/models/submission.py

```diff
@@ -90,7 +109,16 @@ def to_json(self, fields=set()):
         nullable=False,
     )
 
-    project = Column(
+    committed_by = Column(
+        Integer,
+    )
```

Comment:
what's this field for? was trying to find a clue in gdcapi and found [this](https://github.com/NCI-GDC/gdcapi/pull/552/files#diff-8e26e678f358ad346d17d7e63c4e8072R225) ?